### PR TITLE
Solution to challenge 3

### PR DESCRIPTION
**Pull request by tem11010**
Just adding a solution to challenge 3 to make more in line with other challenges.

### CODE REVIEW

File: 06-data-subsetting.Rmd [hunk 1]
```diff
@@ -739,7 +739,7 @@ be changed with the third argument, `drop = FALSE`).
 >
 > ```{r}
 > subset.x <- x[x<7 & x>4]
```
Comment (tomwright01) [inline 06-data-subsetting.Rmd]: Could you change the name of this variable to `x_subset`?

Comment (tem11010) [inline 06-data-subsetting.Rmd]: Sure thing.

**Comment by tomwright01** [comment]
Hi Tim,
Thanks for this, I've made a minor inline comment about changing the name of the variable (I prefer to keep the main variable name at the start and separate operations done on that data with underscores).

Also this pull request now contains using alpha in ggplot with this challenge fix. They don't really go nicely together. I'm not sure how to roll back in this case. The workflow I am using today it to have a master branch that is up to date with swcarpentry/gh-pages, each time I start working on a new issue I'm creating a new branch that I push to my github repo.

`$ git checkout -b IssueXX
 # do modification here
 $ git push origin IssueXX
 # reset for next issue
 $ git checkout master

**Comment by tem11010** [comment]
Thanks Tom. 

That's what I was trying to do. I think I had some issue from merging my changes last week. I rolled back and have merged the changes on a new branch now.

T

**Comment by aammd** [comment]
Thanks @tem11010 , I like this challenge solution (and I am an avowed `alpha` addict). 👍 
After @tomwright01 's comments this looks to be in great shape; merging!

**Pull request merged**